APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -1183,23 +1165,18 @@ - (void) imagePickerControllerDidCancel:(UIImagePickerController *)picker
 
 - (void)taskViewController:(ORKTaskViewController *) __unused taskViewController didFinishWithReason:(ORKTaskViewControllerFinishReason)reason error:(nullable NSError *)error
 {
-    if (reason == ORKTaskViewControllerFinishReasonCompleted)
-    {
+    if (reason == ORKTaskViewControllerFinishReasonCompleted) {
```

Comment:
Yep. In new changes, let's have the else on the same line as the closing bracket. This would be consistent with ResearchKit.

As usual, totally pragmatic. Don't bend over backwards w.r.t. your existing commits.